pkgs/data/misc/cacert/setup-hook.sh

```diff
@@ -1,4 +1,14 @@
-export NIX_SSL_CERT_FILE="${NIX_SSL_CERT_FILE:-@out@/etc/ssl/certs/ca-bundle.crt}"
+if test -r "$NIX_SSL_CERT_FILE"; then
+  # Don't somehow log this more than once, or log if it's irrelevant
+  if [[ "$NIX_SSL_CERT_FILE" != "@out@/etc/ssl/certs/ca-bundle.crt" ]]; then
+    echo "cacert/setup-hook: inheriting existing NIX_SSL_CERT_FILE $NIX_SSL_CERT_FILE" >&2
+  fi
+else
+  if test -n "$NIX_SSL_CERT_FILE"; then
+    echo "cacert/setup-hook: NIX_SSL_CERT_FILE in environment $NIX_SSL_CERT_FILE cannot be read; overriding" >&2
+  fi
+  export NIX_SSL_CERT_FILE="@out@/etc/ssl/certs/ca-bundle.crt"
+fi
```

Comment:
I was thinking of this:

```
export NIX_SSL_CERT_FILE="${NIX_SSL_CERT_FILE:-@out@/etc/ssl/certs/ca-bundle.crt}"
if test ! -r "$NIX_SSL_CERT_FILE"; then
  echo "cacert/setup-hook: NIX_SSL_CERT_FILE in environment $NIX_SSL_CERT_FILE cannot be read; overriding" >&2
  export NIX_SSL_CERT_FILE="@out@/etc/ssl/certs/ca-bundle.crt"
fi
```

It's still a bit of duplication, but at least there's no edge cases around e.g. testing an empty file when NIX_SSL_CERT_FILE isn't set.